packages/govuk-frontend/src/govuk/components/input/input.yaml

```diff
@@ -180,8 +180,6 @@ examples:
     options:
       label:
         text: National Insurance number
-      hint:
-        text: It’s on your National Insurance card, benefit letter, payslip or P60. For example, ‘QQ 12 34 56 C’.
```

Comment:
**question** Thinking that hint might have been there to check where the error would be rendered relative to the hint. By removing it, and not having a test for it, we're no longer checking that it goes 'label, hint, error, field', which I guess is important from a design perspective.

Reply:
I'd be keen to add that hint back and add a test that checks the order of elements inside the `govuk-form-group`, what do you say?

Reply:
There's a separate example which includes both an error message and a hint further down:

https://github.com/alphagov/govuk-frontend/pull/5017/files#diff-64dabf8e89e534dc5f047bb2159938a010e94c221cc75b9b7196d04dcb8a5ceeR451-R461

With a test for the ordering of aria-describedby here:

https://github.com/alphagov/govuk-frontend/pull/5017/files#diff-54a0f41b55758264a220e2eb6e3fba6d181769d8541a05acbe05a1c0963b46bcR241-R261

I think it would make sense to make that example visible though, I'll update it to match this example before I removed the hint.

Reply:
> I'd be keen to add that hint back and add a test that checks the order of elements inside the govuk-form-group, what do you say?

Do you mean the order they appear in the description, or in the DOM?